hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FSOperationType.java

```diff
@@ -45,8 +45,7 @@ public enum FSOperationType {
     SET_OWNER("SO"),
     SET_ACL("SA"),
     TEST_OP("TS"),
-    WRITE("WR"),
-    INIT("IN");
```

Comment:
INIT was introduced in last PR only and was used for the operation to determine HNS nature of account during File System Init.
Later I realized that we already have an operation used for the same purpose introduced in another parallel PR. Since both are fulfilling same purpose, I removed the redundant one.